src/vs/workbench/services/environment/electron-browser/environmentService.ts

```diff
@@ -42,10 +42,10 @@ export class NativeWorkbenchEnvironmentService extends EnvironmentService implem
 	}
 
 	@memoize
-	get webviewResourceRoot(): string { return 'vscode-resource://{{resource}}'; }
+	get webviewResourceRoot(): string { return `${Schemas.vscodeWebviewResource}://{{uuid}}/{{resource}}`; }
```

Comment:
What about the browser environment service?

https://github.com/microsoft/vscode/blob/131acfd129cb62270702eea08980afab1d26d2af/src/vs/workbench/services/environment/browser/environmentService.ts#L205